### PR TITLE
Add a link to the "Uninstall Docker Engine" section

### LINKED ISSUE #33703: Improvement for k8s.io/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd/
As part of "Remove Docker Engine", this page should probably also tell the user that may want to clean up space was consumed by docker, e.g. `/var/lib/docker`.  Otherwise, potentially much space is left used but unmanaged on the node.

### PR DESCRIPTION
**Pull request by JohnTitor**
Fixes #33703

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md [hunk 1]
```diff
@@ -163,3 +163,7 @@ sudo apt-get purge docker-ce docker-ce-cli
 {{% /tab %}}
 {{< /tabs >}}
 
+The above commands don't remove images, containers, volumes, or customized configuration files on your host.
+To delete them, see the [Uninstall Docker Engine](https://docs.docker.com/engine/install/ubuntu/#uninstall-docker-engine) section
+on docker docs.
+Note that it has a risk of deleting containerd, be careful when executing commands on that docs page.
```
Comment (javadoors) [inline content/en/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md:169]:
```suggestion
{{< note >}}
It has a risk of deleting containerd, be careful when executing commands on that docs page.
{{< /note >}}
```

File: content/en/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md [hunk 2]
```diff
@@ -163,3 +163,7 @@ sudo apt-get purge docker-ce docker-ce-cli
 {{% /tab %}}
 {{< /tabs >}}
 
+The above commands don't remove images, containers, volumes, or customized configuration files on your host.
+To delete them, see the [Uninstall Docker Engine](https://docs.docker.com/engine/install/ubuntu/#uninstall-docker-engine) section
+on docker docs.
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md:168]:
```suggestion
within the documentation for Docker.
```

File: content/en/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md:169]: I'd actually use a `{{< caution >}}` here as the consequences for doing the wrong thing on every node could be rather unfortunate.

**Comment by JohnTitor** [comment]
Thanks both, addressed them.

File: content/en/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md [hunk 3]
```diff
@@ -163,3 +163,10 @@ sudo apt-get purge docker-ce docker-ce-cli
 {{% /tab %}}
 {{< /tabs >}}
 
+The above commands don't remove images, containers, volumes, or customized configuration files on your host.
+To delete them, see the [Uninstall Docker Engine](https://docs.docker.com/engine/install/ubuntu/#uninstall-docker-engine) section
+within the documentation for Docker.
+
+{{< caution >}}
+It has a risk of deleting containerd, be careful when executing commands on that docs page.
+{{< /caution >}}
```
Comment (zacharysarah) [inline content/en/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md:172]: Some edits for clearer language:
```suggestion
The preceding commands don't remove images, containers, volumes, or customized configuration files on your host.
To delete them, follow Docker's instructions to [Uninstall Docker Engine](https://docs.docker.com/engine/install/ubuntu/#uninstall-docker-engine).

{{< caution >}}
Docker's instructions for uninstalling Docker Engine create a risk of deleting containerd. Be careful when executing commands.
{{< /caution >}}
```

**Review by zacharysarah — CHANGES_REQUESTED** [review]
@JohnTitor Thanks for this. ✨ Some edits for clearer language, otherwise LGTM

**Comment by JohnTitor** [comment]
Applied, thanks :)

**Pull request merged**